### PR TITLE
Port library and test code to new 0.6 names to reduce warning count.

### PR DESCRIPTION
**Pull request by cgranade**
This PR updates library and test code to use 0.6 names and syntax in order to reduce the number of warnings generated during builds.

### CODE REVIEW

File: Standard/src/Arithmetic/Arithmetic.qs [hunk 1]
```diff
@@ -32,22 +32,6 @@ namespace Microsoft.Quantum.Arithmetic {
         controlled adjoint auto;
     }
 
-    /// # Summary
```
[the remark this answers is not in the document]
Comment (cgranade) [reply Standard/src/Arithmetic/Arithmetic.qs]: That's correct, I went on and removed it as a part of cleaning up other issues.

File: Standard/src/Arithmetic/Modular.qs [hunk 2]
```diff
@@ -26,15 +27,15 @@ namespace Microsoft.Quantum.Arithmetic {
     /// Integer y in `LittleEndian` format that `increment` a is added to.
     ///
     /// # See Also
```
[the remark this answers is not in the document]
Comment (cgranade) [reply Standard/src/Arithmetic/Modular.qs]: Good point, we should follow up with @alan-geller on resolution for `/// # See Also`.

File: Standard/src/Arithmetic/Modular.qs [hunk 3]
```diff
@@ -64,89 +65,49 @@ namespace Microsoft.Quantum.Arithmetic {
     /// For the circuit diagram and explanation see Figure 5 on [Page 5
     /// of arXiv:quant-ph/0205095v3](https://arxiv.org/pdf/quant-ph/0205095v3.pdf#page=5).
     operation IncrementPhaseByModularInteger(increment : Int, modulus : Int, target : PhaseLittleEndian) : Unit {
```
[the remark this answers is not in the document]
Comment (cgranade) [reply Standard/src/Arithmetic/Modular.qs]: Ah, cool, I didn't realize that was implemented yet, thanks for pointing that out!

**Comment by bettinaheim** [comment]
I approved the PR - I am aware that this PR will temporarily break things, we've talked about it, and it seems reasonable to do so to bypass a circular dependency. The broken things will thus be fixed in subsequent steps.

**Pull request merged**